app/routes/events/view/team/permissions.ts

```diff
@@ -87,65 +87,65 @@ export default class EventsViewTeamPermissions extends Route.extend({
           organizer : true
```

Comment:
also this one

Reply:
done

Reply:
can u pls give me the git command of upstream